File: 151002/Kragel/dc/src/main/java/by/bsuir/dc/entity/Creator.java
package by.bsuir.dc.entity;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import lombok.Value;

@Getter
@Setter
public class Creator extends AbstractEntity {
    private String login;
    private String password;
    private String firstName;
    private String lastName;
}

File: 151004/Buhovets/lab1/REST/Entity/Db/Creator.cs
using REST.Entity.Common;
using System.ComponentModel.DataAnnotations;

namespace REST.Entity.Db
{
    public class Creator(string login, string password, string firstName, string lastName) : AbstractEntity
    {
        [MinLength(2)]
        public string Login { get; set; } = login;
        [MinLength(8)]
        public string Password { get; set; } = password;
        [MinLength(2)]
        public string FirstName { get; set; } = firstName;
        [MinLength(2)]
        public string LastName { get; set; } = lastName;
        public ICollection<Tweet> Tweets { get; set; } = [];


        public Creator(int id, string login, string password, string firstName, string lastName) :
            this(login, password, firstName, lastName)
        {
            Id = id;
        }

        public Creator() : this(string.Empty, string.Empty, string.Empty, string.Empty) { }
    }
}
